roaring/roaring.go

```diff
@@ -4429,6 +4726,75 @@ func xorBitmapRun(a, b *Container) *Container {
 	return output
 }
 
+// CompareEquality is used mostly in test cases to confirm that two bitmaps came
+// out the same. It does not expect corresponding opN, or OpWriter, but expects
+// identical bit contents. It does not expect identical representations; a bitmap
+// container can be identical to an array container. It returns a boolean value,
+// and also an explanation for a false value.
+func (b *Bitmap) BitwiseEqual(c *Bitmap) (bool, error) {
+	biter, _ := b.Containers.Iterator(0)
+	citer, _ := c.Containers.Iterator(0)
```

Comment:
any reason we're using the Containers Iterator instead of the bitmap iterator?

Reply:
I didn't think of it. Although I think I prefer comparing by containers, because the existence of an unmatched non-empty container is a simple answer, and container equality is reasonably easy to check.